### PR TITLE
Add shell script to gather log data via SSM

### PR DESCRIPTION
**Pull request by dav3r**
#  #

## 🗣 Description ##

This PR adds an "extras" directory containing a shell script that can be used to fetch the ClamAV scan log data from a list of AWS EC2 instances via [SSM](https://docs.aws.amazon.com/systems-manager/latest/userguide/what-is-systems-manager.html).

## 💭 Motivation and context ##

This addresses the issue raised in #30.  I will leave it up to discussion whether or not this PR is enough for us to close out #30.

## 🧪 Testing ##

I ran the script several times and verified that it worked as expected.  I also tested the script with no arguments and with `-h` and with `--help` and confirmed that the usage text was displayed as intended.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [x] Finalize version.

## ✅ Post-merge checklist ##

- [x] Create a release.

### CODE REVIEW

File: extras/clamav_log_report.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# Gather ClamAV scan log data from a list of AWS instances via SSM.
+#
+# Usage: ./clamav_log_report.sh <instance-id>...
+
```
Comment (jsf9k) [inline extras/clamav_log_report.sh:6]: Is there any reason not to include our usual foo here?
```suggestion

set -o nounset
set -o errexit
set -o pipefail

```

**Review by jsf9k — CHANGES_REQUESTED** [review]

File: extras/clamav_log_report.sh (hunk 1 above)
Comment (dav3r) [reply extras/clamav_log_report.sh:6]: Nope- good catch, I just forgot about them, thanks!

**Review by jsf9k — APPROVED** [review]

File: extras/clamav_log_report.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# tee -i: Ignore SIGINT signals
+exec > >(tee -ai "$logfile")
+exec 2> >(tee -ai "$logfile" >&2)
+
+instances=("$@")
+
+for instance_id in "${instances[@]}"; do
+  aws ssm start-session --target="$instance_id" \
+    --document=AWS-StartInteractiveCommand \
+    --parameters="command='hostname; tail --lines=12 /var/log/clamav/lastscan.log'"
```
Comment (mcdonnnj) [inline extras/clamav_log_report.sh]: Is there any documentation for the `tail --lines=12 /var/log/clamav/lastscan.log` command we're sending? Also a minor thing but thoughts on breaking `/var/log/clamav/lastscan.log` out into a variable?

Comment (dav3r) [reply extras/clamav_log_report.sh]: Good suggestions, thanks.  I added a comment in https://github.com/cisagov/clamav-report/pull/31/commits/5f20764eeb16d89560edcf0df36a9e1515d0177e and made the filename a variable in https://github.com/cisagov/clamav-report/pull/31/commits/d7b480678e89ff13c9771b44c1a8892f30bec92d.  Let me know what you think.

**Review by mcdonnnj — APPROVED** [review]
Seems like a reasonable path to provide this functionality. I will say, for the sake of discussion, that I don't think this resolves #30 as I would like native support in the package. We may want to explore offerings like [community.aws.aws_ssm connection](https://docs.ansible.com/ansible/latest/collections/community/aws/aws_ssm_connection.html) to support this kind of connectivity natively.

**Comment by dav3r** [comment]
> I will say, for the sake of discussion, that I don't think this resolves #30 as I would like native support in the package. We may want to explore offerings like [community.aws.aws_ssm connection](https://docs.ansible.com/ansible/latest/collections/community/aws/aws_ssm_connection.html) to support this kind of connectivity natively.

I'm fine with leaving #30 open.  For what it's worth, I played around with [`community.aws.aws_ssm connection`](https://docs.ansible.com/ansible/latest/collections/community/aws/aws_ssm_connection.html) in the past and couldn't get it to work as I had hoped here, but it looks like there have been some updates since then.  I updated the description in #30 to mention `community.aws.aws_ssm connection` in the "Implementation notes" section.

PS: I believe the issue was that `community.aws.aws_ssm connection` did not allow me to supply the `--parameters` data when attempting to start an SSM session.

**Pull request merged**